### PR TITLE
fix(sdk) Update README.md for SDK

### PR DESCRIPTION
**Pull request by albertoperdomo**
- Warning about React 17
- Update supported and planned features

### CODE REVIEW

File: enterprise/frontend/src/embedding-sdk/README.md [hunk 1]
```diff
@@ -19,20 +19,24 @@ Features currently supported:
 - embedding dashboards - static
 - embedding dashboards - w/drill-down
 - embedding the collection browser
+- ability for the user to modify existing questions
 - theming with CSS variables
-- plugins for custom actions
+- plugins for custom actions, overriding dashboard card menu items
+- subscribing to events
```
Comment (heypoom) [inline enterprise/frontend/src/embedding-sdk/README.md:25]: What kind of "events" are we referring to here? If we're referring to dashboard loading events, then this is already supported in the latest release.

**Review by heypoom — APPROVED** [review]
Nice, thanks!

File: enterprise/frontend/src/embedding-sdk/README.md (hunk 1 above)
Comment (albertoperdomo) [reply enterprise/frontend/src/embedding-sdk/README.md:25]: Yes, that's why it's listed under features supported. Now?

Comment (heypoom) [reply enterprise/frontend/src/embedding-sdk/README.md:25]: @albertoperdomo oh sorry, I thought this was in the "planned" section - the GitHub diffs are a bit misleading. all good then.

**Pull request merged**